tests/test_smile.py

```diff
@@ -32,7 +31,7 @@
 # as inclusion point
 
 
-class TestPlugwise:
+class TestPlugwise:  # pylint: disable=attribute-defined-outside-init
```

Comment:
Glanced most of this, we've checked eachother on this with the quick branch-checks and tests looking good!